packages/vue/jest.config.cjs

```diff
@@ -13,10 +13,10 @@ module.exports = {
   ],
   coverageThreshold: {
     global: {
-      branches: 62,
-      functions: 81,
-      lines: 87,
-      statements: 87,
+      branches: 89,
+      functions: 90,
+      lines: 93,
+      statements: 93,
```

Comment:
These will be the final coverage level at the end of this refactor. 

Primitives can still use more unit tests, which would increase the coverage even more. Primitives are outside the scope of this refactor however.

Reply:
This is way higher what I thought we could hit in this amount of time! Thank you @wlee221!!!

Reply:
Whoa whatttt 🤯

Reply:
This should be good enough. Going higher is better but may not worth the effort 😓